SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Reporting Security Issues
+
+We at Snyk value the security community and believe that responsible disclosure of security vulnerabilities in open source packages helps us ensure the security and privacy of the users.
+
+If you believe you have found a security vulnerability on Snyk, we encourage you to let us know right away. We will investigate all legitimate reports and do our best to quickly fix the problem. Before reporting though, please review our responsible disclosure policy, and those things that should not be reported.
+
+Submit your report to security@snyk.io (one issue per report) and respond to the report with any updates. Please do not contact employees directly or through other channels about a report.
+
+Report security bugs in third-party modules to the person or team maintaining the module. You can also report a vulnerability through our [Snyk Vulnerability Disclosure](https://docs.google.com/a/snyk.io/forms/d/e/1FAIpQLSemwgWZ0JgK1ZULKhy9DZCQ5KulbLEldvmokAuRtt-_nrqNlA/viewform) program.
+
+### Responsible Disclosure Policy
+
+We ask that:
+
+- You give us reasonable time to investigate and mitigate an issue you report before making public any information about the report or sharing such information with others.
```

Comment:
<label>**nitpick**</label> [non-blocking]: <subject>Are we able to quntify how long is a reasonable amount of time? (i.e. a month?) or do we prefer to leave this open as we will have different priorities</subject>

Reply:
A good question, this policy is taken from the CLI repo, this may be something that the owners of this repo can be more specific about.